src/main/java/EntityInterfaces/RecipeItem.java

```diff
@@ -0,0 +1,28 @@
+package EntityInterfaces;
+
+import Entities.Ingredient;
+
+public interface RecipeItem {
+    /**
+     * returns the ingredient for  the recipeitem.
+     * @return returns the ingredient for  the recipeitem.
+     *
+     * returns a float of the quantity needed for the recipe.
+     * @return returns a float of the quantity needed for the recipe.
+     *
+     * displays the recipeitem in string form.
+     * @return a string of the recipeitem.
+     */
+    Ingredient ingredient();
+    float quantity();
+    boolean optional();
```

Comment:
Don't put comments together like this. Make one for each.

Reply:
This wasn't my code but can def fix it.